scripts/backend_test_shards.json

```diff
@@ -252,6 +252,8 @@
         "jobs.transforms.config_validation_test",
         "jobs.transforms.collection_validation_test",
         "jobs.transforms.question_validation_test",
+        "jobs.transforms.skill_validation_test",
+        "jobs.transforms.story_validation_test",
```

Comment:
Can these new tests be moved to shard 5?

Reply:
Yes, done PTAL